src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java

```diff
@@ -72,6 +72,7 @@
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
why additional import?

Reply:
it is not an additional import, it was missing, i have no idea how it was being compiled.